release_notes/ose_3_1_release_notes.adoc

```diff
@@ -175,4 +194,11 @@ Features Support Scope]
 
 The following features are in Technology Preview:
 
-- Binary builds, and the `*Dockerfile*` source type for builds.
+* Binary builds, and the Dockerfile source type for builds.
+* Support for OpenShift Enterprise running on Atomic Host.
+* Containerized installations, meaning all OpenShift Enterprise components running in a container.
+
+== Known Issues
+
+* When pushing an internal registry when multiple registries share the same NFS volume, there is a chance the push will fail. A work-around has been suggested.
```

Comment:
The workaround is #1174 

I was waiting for this to merge, but it's been a while. You and Paul Weil commented on (what looks like) your Friday, but I'll comment again. Either way, I was thinking this could link out to that, but as it's not merged yet, I got nothing to link to. I'll keep an eye on it.